Tasks/app-store-release/task.json

```diff
@@ -13,7 +13,7 @@
     "demands": [ "xcode" ],
     "version": {
         "Major": "1",
-        "Minor": "115",
+        "Minor": "116",
         "Patch": "0"
```

Comment:
Can you please apply any changes you make based on comments for `app-store-promote/task.json` to `app-store-promote/task.json`?